### PR TITLE
sfsexp 1.4.1 (new formula)

### PR DESCRIPTION
**Pull request by hgvhgv**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

Symbolic-expression library for C and C++. Required by notmuch for its notmuch-git Python script, which I would like to add to homebrew's notmuch bottle (see discussion here:
https://github.com/orgs/Homebrew/discussions/5204). 

`brew audit --new sfsexp` says "GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)" but I believe it's worth including for the notmuch-git script to be in the notmuch bottle. Otherwise `brew audit --new` passes.

### CODE REVIEW

File: Formula/s/sfsexp.rb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    system "./configure", "--disable-silent-rules", *std_configure_args(prefix: prefix)
+    system "make"
+    system "make", "install"
+    system "doxygen"
+    doc.mkpath
+    man3.mkpath
+    libexec.mkpath
+    doc.install Dir["doxygen/html/*"]
+    man3.install "doxygen/man/man3/IO.3" => "sexp_I0.3"
+    man3.install Dir["doxygen/man/man3/{[!IO.3]}*"]
```
Comment (SMillerDev) [inline Formula/s/sfsexp.rb:24]: If make doesn't do this I don't think we should

File: Formula/s/sfsexp.rb [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  license "LGPL-2.1-or-later"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "doxygen" => :build
+  depends_on "libtool" => :build
+  uses_from_macos "m4" => :build
+
+  def install
+    system "./configure", "--disable-silent-rules", *std_configure_args(prefix: prefix)
```
Comment (SMillerDev) [inline Formula/s/sfsexp.rb:15]:
```suggestion
    system "./configure", "--disable-silent-rules", *std_configure_args
```

File: Formula/s/sfsexp.rb (hunk 1 above)
Comment (hgvhgv) [reply Formula/s/sfsexp.rb]: removed (and removed doxygen dependency)

**Comment by hgvhgv** [comment]
> Thanks for contributing to Homebrew! 🎉 It looks like you're having trouble with a CI failure. See our [contribution guide](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md) for help. You may be most interested in the section on [dealing with CI failures](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md#dealing-with-ci-failures). You can find the CI logs in the [Checks tab](https://github.com/Homebrew/homebrew-core/pull/200568/checks) of your pull request.

`brew audit --new` fails because the sfsexp github isn't notable enough ("GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars"). This library would be a dependency of notmuch, which is already in homebrew, so I think it would qualify for inclusion. Is there a way to get it to proceed beyond this?

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@hgvhgv can you squash the commits and force update the PR? thanks!

**Review by daeho-ro — APPROVED** [review]

**Comment by chenrui333** [comment]
@hgvhgv, thanks for your contribution to Homebrew-core! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request merged**